src/components/ConfirmStatusModal/ConfirmStatusModal.js

```diff
@@ -21,6 +21,7 @@ class ConfirmStatusModal extends React.Component {
     message: PropTypes.node.isRequired,
     onConfirm: PropTypes.func.isRequired,
     open: PropTypes.bool.isRequired,
+    onCancel: PropTypes.func,
```

Comment:
If this property required you should mark it as `isRequired` in other case it should be listed in defautProps with default value